pkgs/by-name/li/limine/package.nix

```diff
@@ -0,0 +1,59 @@
+# Builds limine with all available features.
+
+{
+  # Helpers
+  clangStdenv
+, fetchurl
+, lib
+, # Dependencies
+  llvmPackages
+, mtools
+, nasm
+}:
+
+let
+  version = "7.1.0";
+in
+clangStdenv.mkDerivation {
+  inherit version;
+  pname = "limine";
+  src = fetchurl {
+    url = "https://github.com/limine-bootloader/limine/releases/download/v${version}/limine-${version}.tar.gz";
```

Comment:
Why do we want to use the release asset instead of the downloading the source code?

Reply:
We don't use the Git source but the release tarball, as the source has a
`./bootstrap` script performing non-standard network access to download
resources (a custom Git submodule-like replacement). Packaging this in Nix
is very cumbersome compared to using the release tarball, where the
`./bootstrap` step was already executed, and we can `./configure` right away.
  

I added a comment to reflect that

Reply:
See https://github.com/NixOS/nixpkgs/pull/298048/files#r1539841844